koku/masu/processor/tasks.py

```diff
@@ -309,3 +317,25 @@ def update_charge_info(schema_name, provider_uuid, start_date=None, end_date=Non
 
     updater = ReportChargeUpdater(schema_name, provider_uuid)
     updater.update_charge_info(start_date, end_date)
+
+
+@app.task(name='masu.processor.tasks.refresh_materialized_views', queue_name='reporting')
+def refresh_materialized_views(schema_name, provider_type, manifest_id=None):
+    """Refresh the database's materialized views for reporting."""
+    materialized_views = ()
+    if provider_type in (Provider.PROVIDER_AWS, Provider.PROVIDER_AWS_LOCAL):
+        materialized_views = AWS_MATERIALIZED_VIEWS
+    with schema_context(schema_name):
+        for view in materialized_views:
+            table_name = view._meta.db_table
+            with connection.cursor() as cursor:
+                LOG.info(f'Refreshing {table_name}.')
+                cursor.execute(
+                    'REFRESH MATERIALIZED VIEW CONCURRENTLY {}'.format(table_name)
```

Comment:
my consistency bells are ringing
```suggestion
                    f'REFRESH MATERIALIZED VIEW CONCURRENTLY {table_name}'
```